contracts/KeyValuePair.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: MIT
+pragma solidity =0.8.19;
+
+import "./interfaces/IKeyValuePair.sol";
+
+/**
+ * A simple contract to log key/value pair events for the calling address.
+ */
+contract KeyValuePair is IKeyValuePair {
+
+    /** @inheritdoc IKeyValuePair*/
+    function updateValue(string memory _key, string memory _value) external {
```

Comment:
Should we make this function receive an array of keys and values? So that it could emit an arbitrary number of keys and values at once?